src/components/entity/list.vue

```diff
@@ -142,11 +144,49 @@ export default {
       })
     });
 
+    const creatorIds = useQueryRef({
+      fromQuery: (query) => {
+        const stringIds = arrayQuery(query.submitterId, {
+          validator: (value) => /^[1-9]\d*$/.test(value)
+        });
+        return stringIds.length !== 0
+          ? stringIds.map(id => Number.parseInt(id, 10))
+          : (entityCreators.dataExists ? [...entityCreators.ids] : []);
+      },
+      toQuery: (value) => ({
+        creatorId: value.length === entityCreators.length
+          ? []
+          : value.map(id => id.toString())
+      })
+    });
+    watch(() => entityCreators.dataExists, () => {
+      if (creatorIds.value.length === 0 && entityCreators.length !== 0)
+        creatorIds.value = [...entityCreators.ids];
+    });
+
+    const creationDateRange = useQueryRef({
+      fromQuery: (query) => {
+        if (typeof query.start === 'string' && typeof query.end === 'string') {
+          const start = DateTime.fromISO(query.start);
+          const end = DateTime.fromISO(query.end);
+          if (start.isValid && end.isValid && start <= end)
+            return [start.startOf('day'), end.startOf('day')];
+        }
+        return [];
+      },
+      toQuery: (value) => (value.length !== 0
+        ? { start: value[0].toISODate(), end: value[1].toISODate() }
+        : { start: null, end: null })
+    });
```

Comment:
Since this is the same as `SubmissionList`, could we make it reusable? I expect that we'll use it other places too. Maybe we add a function `useDateQuery()`.